shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/BaseTypes.hs

```diff
@@ -316,16 +316,15 @@ activeSlotLog f = (fromIntegral $ unActiveSlotLog f) / fpPrecision
 data Globals = Globals
   { epochInfo :: !(EpochInfo Identity)
   , slotsPerKESPeriod :: !Word64
+    --
+    --   The nonce leakage parameter τ from the paper
+    --   "Ouroboros praos: An adaptively-secure, semi-synchronous proof-of-stake protocol".
+    --   This value is used in a number of places; for example, protocol updates
```

Comment:
"This value is used ..." is a bit ambiguous, sounds like it could be making a claim about how the paper uses it.

Reply:
good point! I will say "We use `stabilityWindow` in a number of places..."